docs/.vuepress/config.js

```diff
@@ -4,8 +4,17 @@ const examples = require('./containers/examples');
 const sourceCodeLink = require('./containers/sourceCodeLink');
 const nginxRedirectsPlugin = require('./plugins/generate-nginx-redirects');
 const assetsVersioningPlugin = require('./plugins/assets-versioning');
+const { getBuildDocsVersion, getLatestVersion } = require('./helpers');
 
 const buildMode = process.env.BUILD_MODE;
+const version = getBuildDocsVersion() || '**';
+const isLatestOrMultiVersion = getBuildDocsVersion() === getLatestVersion() || !getBuildDocsVersion();
+const base = isLatestOrMultiVersion ? '/docs/' : `/docs/${version}/`;
+const redirectsPlugin = isLatestOrMultiVersion ?
```

Comment:
I see that the plugin `redirectsPlugin` cannot be added conditionally as right now VuePress prints a warning when an empty array is passed
![image](https://user-images.githubusercontent.com/571316/133569742-5ba6dab8-1233-400b-8991-17167a6aa62d.png)

I think that the `plugin` config key has to be extracted from the config object and then the redirect plugin can be added by `.push()` call.

Reply:
done in 219ea49. It was enough to change `[]` to `{}` as a plugin.